bigbone/src/test/kotlin/social/bigbone/api/method/AppMethodsTest.kt

```diff
@@ -20,7 +20,7 @@ class AppMethodsTest {
         val application = appMethods.createApp(
             clientName = "bigbone-sample-app",
             redirectUris = TestConstants.REDIRECT_URI,
-            scope = Scope(Scope.Name.ALL)
+            scope = Scope(Scope.Name.READ, Scope.Name.WRITE, Scope.Name.PUSH)
```

Comment:
This could also use `Scope.FULL_SCOPE` if we follow the companion object route

Reply:
I guess something like this would also make sense for the admin scope?